setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 ignore =
-    E501,F403,F405,E226,E265,E731,E402,E266,
+    E501,F403,F405,E226,E265,E731,E402,E266,F999,
```

Comment:
``
flake8 --version
3.2.1 (pyflakes: 1.3.0, mccabe: 0.5.2, pycodestyle: 2.2.0) CPython 2.7.12 on Linux
``

I think with flake8 you can also have a global config file wherein you state all of the options that you want to be ignored in *any* project -- but maybe that's not your case